packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -1000,6 +1012,8 @@ class ImportedBucket extends BucketBase {
   }
 
   private generateBucketWebsiteUrl() {
-    return `${this.bucketName}.s3-website-${new cdk.Aws().region}.amazonaws.com`;
+    return this.bucketWebsiteNewUrlFormat
+      ? `${this.bucketName}.s3-website-${cdk.Stack.find(this).region}.${cdk.Stack.find(this).urlSuffix}`
```

Comment:
It is just the other way around; new format is with `.`, old format is with `-`.

I support the default being `false`.